distribution/lib/Standard/Base/0.0.0-dev/src/Data/Numbers.enso

```diff
@@ -348,7 +368,8 @@ type Number
     nan : Decimal
     nan = Double.NaN
 
-    ## Checks if the given number is the floating-point Not-a-Number value.
+    ## GROUP Logical
```

Comment:
Hmmm, wouldn't `is_nan` fit better into `Math` group?

It feels very similar to e.g. `is_infinite` that is there.

I assume a parallelism to `is_nothing`, but IMO `is_nothing` is a bit more special (and does not have a much better category anyway), whereas `is_nan` is rather closely related to the Math operations.

But that is still just my subjective suggestion, I think `Logical` seems OK too.